config/prod/htan-dcc-chop.yaml

```diff
@@ -0,0 +1,29 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html) that will
+# work with the s3-synapse-sync lambda: https://github.com/Sage-Bionetworks/s3-synapse-sync
+template_path: "remote/s3-synapse-sync-bucket.j2"
+stack_name: "htan-dcc-chop"
+stack_tags:
+  Department: "CompOnc"
+  Project: "HTAN"
+  OwnerEmail: "xengie.doan@sagebase.org"
+hooks:
+  before_launch:
+    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/s3-synapse-sync/master/s3-synapse-sync-bucket.j2 --create-dirs -o templates/remote/s3-synapse-sync-bucket.j2"
+dependencies:
+  - "prod/s3-synapse-sync.yaml"
+parameters:
+  BucketName: "htan-dcc-chop"  # must match bucket name in data/s3-synapse-sync-bucket-vars.yaml
```

Comment:
i think the comment on this line is outdated, `data/s3-synapse-sync-bucket-vars.yaml` no longer exists.

would be good to remove that comment from the documentation  for creating this bucket at https://github.com/Sage-Bionetworks/s3-synapse-sync#create-buckets

Reply:
I was able to find this file in prod: https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/master/config/prod/s3-synapse-sync.yaml

Reply:
doesn't matter, it's still outdated.  it originated from the documentation.  it would be good to remove this comment from all the existing ones in prod as well.